serverless_llm/store/serverless_llm_store/transformers.py

```diff
@@ -221,15 +221,19 @@ def best_effort_load(
     device_map: DeviceMapType = "auto",
     torch_dtype: Optional[torch.dtype] = None,
     storage_path: Optional[str] = None,
-):
+):  
     client = SllmStoreClient("localhost:8073")
     ret = client.load_into_cpu(model_path)
     if not ret or ret == False:
         raise ValueError(f"Failed to load model {model_path} into CPU")
 
-    replica_uuid = _get_uuid()
+    replica_uuid = _get_uuid()   
     device_map = _transform_device_map_to_dict(device_map)
 
+    if isinstance(device_map, dict):
```

Comment:
what about 'str'?

Reply:
<img width="562" alt="image" src="https://github.com/user-attachments/assets/c4d2e791-4757-4469-b2c5-3199b0e2709b">
According to the code snippet above, the return device_map will be str when the input device map is in ["auto",  "balanced", "balanced_low_0", "sequential"]. Otherwise, it will be turned into dict.